controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -3813,4 +3815,45 @@ OperationFailedRuntimeException capabilityAlreadyRegisteredInContext(String capa
     @Message(id = 518, value = "Resource /extension=%s uses a non-canonical module name; use the canonical form %s. " +
             "The canonical representation includes slot information in the module name only if the is slot is not 'main'.")
     String nonCanonicalExtensionName(String nonCanonical, String canonical);
+
+    @LogMessage(level = WARN)
+    @Message(id = 519, value = "Attribute '%2$s' of element '%1$s' is no longer supported and will be ignored")
+    void attributeIgnored(QName elementName, QName attributeName);
```

Comment:
Optional. Maybe it is easier / clearer if instead of specifying the argument number, the argument methods are put in orden as they are expected in the log message

Reply:
What aspect of argument indices is not clear?

Frankly, we should probably always use argument indices in the message text, since the text itself can always change, not to mention that the incident order may change during translation.

In this instance, I find it more natural to specify the element name before the attribute name in the method signature (just as you would encounter them in the XML itself).

Reply:
> What aspect of argument indices is not clear?

Well, it is that it just results more readable since none of the arguments for the loggers use the positional notation; they use the orden that are defined. Anyway, I commented about it as an optional change, but for sure, it is ok to leave it as it is.